cli/operator/node.go

```diff
@@ -288,29 +290,34 @@ func setupOperatorStorage(logger *zap.Logger, db basedb.IDb) (operatorstorage.St
 	return nodeStorage, operatorData
 }
 
-func setupSSVNetwork(logger *zap.Logger) (beaconprotocol.Network, forksprotocol.ForkVersion) {
-	if len(cfg.P2pNetworkConfig.NetworkID) == 0 {
-		cfg.P2pNetworkConfig.NetworkID = format.DomainType(types.GetDefaultDomain()).String()
-	} else {
-		// we have some custom network id, overriding default domain
-		domainType, err := format.DomainTypeFromString(cfg.P2pNetworkConfig.NetworkID)
-		if err != nil {
-			logger.Fatal("failed to parse network id", zap.Error(err))
-		}
-		types.SetDefaultDomain(spectypes.DomainType(domainType))
+func setupSSVNetwork(logger *zap.Logger) (beaconprotocol.Network, forksprotocol.ForkVersion, error) {
+	beaconNetwork, err := networkconfig.GetNetworkByName(cfg.P2pNetworkConfig.NetworkID)
```

Comment:
I'm not sure it's good that we'll use `cfg.P2pNetworkConfig.NetworkID` as the config param for network name.
Network name is the name of the network for the configuration.
NetworkID was originally used in P2P to make sure nodes from different networks won't try to connect each other.
So for sure we want the Network name to be on higher level in the config than P2P since now it represents more than what it used to. 
The network configuration should include a NetworkID that is a byte that is used for the same purpose as the previous NetworkID.

Reply:
> So for sure we want the Network name to be on higher level in the config than P2P since now it represents more than what it used to.

okay, I agree

> The network configuration should include a NetworkID that is a byte that is used for the same purpose as the previous NetworkID.

Why do we need to keep it in the config file if it's configured in the network config?

Reply:
done